src/View/Helper/GooglePlacesHelper.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Geo\View\Helper;
+
+/**
+ * This is a CakePHP helper that helps users to integrate Google Places
+ * into their application by only writing PHP code. This helper depends on jQuery.
+ *
+ * CodeAPI: http://code.google.com/intl/de-DE/apis/maps/documentation/javascript/basics.html
+ * Icons/Images: http://gmapicons.googlepages.com/home
+ *
+ * @author Michael Heit
+ * @link http://www.dereuromark.de/2010/12/21/googlemapsv3-cakephp-helper/
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ * @property \Cake\View\Helper\HtmlHelper $Html
+ * @property \Cake\View\Helper\FormHelper $Form
+ */
+class GooglePlacesHelper extends GoogleMapHelper {
+
+	/**
+	 * Needed helpers
+	 *
+	 * @var array
+	 */
+	public $helpers = ['Form', 'Html'];
+
+	/**
+	 * Wrapper function for control like Form->input
+	 *
+	 * @param string $fieldName name of input field
+	 * @param array $fieldOptions associative array of settings are passed. Should be the same as uses on Form->control
+	 * @param array $googleOptions associative array of settings for places.Autocomplete
+	 *
+	 * @return string divContainer
+	 */
+	public function input($fieldName, array $fieldOptions = [], array $googleOptions = []) {
+		return $this->control($fieldName, $fieldOptions, $googleOptions);
+	}
+
+	/**
+	 * This the initialization point of the script
+	 * Returns the div container you can echo on the website
+	 *
+	 * @param string $fieldName name of input field
+	 * @param array $fieldOptions associative array of settings are passed. Should be the same as uses on Form->control
+	 * @param array $googleOptions associative array of settings for places.Autocomplete
+	 *
+	 * @return string divContainer
+	 */
+	public function control($fieldName, array $fieldOptions = [], array $googleOptions = []) {
+		$id = isset($fieldOptions['id']) && $fieldOptions['id'] != '' ? $fieldOptions['id'] : $fieldName;
```

Comment:
!empty(fieldOptions['id']) is shorter than both checks. 0 is not a valid id anyway right?